cymetric/metrics.py

```diff
@@ -274,24 +274,24 @@ def decommission_series(entry, exit):
 
 
 @metric(name='Agents', depends=_agentsdeps, schema=_agentsschema)
-def agents(entry, exit, decom, info):
+def agents(entry, agent_exit, decom, info):
     """Computes the Agents table. This is tricky because both the AgentExit
     table and the DecomSchedule table may not be present in the database.
     Furthermore, the Info table does not contain the AgentId column. This
     computation handles the calculation of the ExitTime in the face a
     significant amounts of missing data.
     """
     mergeon = ['SimId', 'AgentId']
-    ent = tools.raw_to_series(entry, ['SimId', 'AgentId'], 'Kind')
-    idx = ent.index
+    idx = ['SimId','AgentId','ExitTime']
     df = entry[['SimId', 'AgentId', 'Kind', 'Spec', 'Prototype', 'ParentId',
                 'Lifetime', 'EnterTime']]
-    if exit is None:
-        agent_exit = pd.Series(index=idx, data=[np.nan] * len(idx))
+    if agent_exit is None:
+        agent_exit = entry[['SimId','AgentId']]
```

Comment:
```suggestion
        agent_exit = entry[['SimId','AgentId']].copy()
```
Pandas seems to advices this usage instead.